lib/mk-soxin-module.nix

```
@@ -13,6 +13,8 @@ in
 , name
 , includeKeyboardLayout ? false
 , includeTheme ? false
+, includeTool ? false
+, includeprogrammingLanguage ? false
```

```suggestion
, includeProgrammingLanguage ? false
```

Shouldn't it be a camelcase?